CLAUDE.md

```
@@ -0,0 +1,95 @@
+
+### Running E2E Tests Locally
+
+E2E tests use the `osde2e` build tag and are excluded from unit test runs. To run against a cluster:
+
+```bash
+OCM_ENVIRONMENT=stage KUBECONFIG=/path/to/kubeconfig ginkgo --tags=osde2e -v test/e2e/
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)